nano.save

```diff
@@ -0,0 +1 @@
+
```

Comment:
FYI: this is probably a temporary file used by Nano editor in order to prevent lost due to electrical shortage or something similar. Whenever you see something like this popping up the `git commit` window, just add the file (or the extension of the file with `*.save` in this case) to your `.gitignore` file. By doing that you prevent this from happening in the future.